src/Domain/Staff/StaffAbsenceGateway.php

```diff
@@ -173,7 +173,7 @@ public function queryApprovedAbsencesByDateRange(QueryCriteria $criteria, $dateS
             ->leftJoin('gibbonPerson AS coverage', 'gibbonStaffCoverage.gibbonPersonIDCoverage=coverage.gibbonPersonID')
             ->where('gibbonStaffAbsenceDate.date BETWEEN :dateStart AND :dateEnd')
             ->where("gibbonStaffAbsence.status = 'Approved'")
-            ->where("gibbonPerson.status = 'Full'")
+            //->where("gibbonPerson.status = 'Full'")
```

Comment:
It's okay to remove this line, rather than commenting it out. We'll never remember why it was left there in the future, and if we do ever need it, all the past code is in the git commit history : )